apps/web/src/components/quick-start/in-app-onboarding/TriggerNode.tsx

```diff
@@ -80,6 +81,7 @@ function TriggerButton({ setOpened }: { setOpened: (value: boolean) => void }) {
     async function createOnBoardingTemplate() {
       const payloadToCreate = {
         notificationGroupId: groups[0]._id,
+        isBlueprint: false,
```

Comment:
do we have to set it?

Reply:
Added this flag to make sure that there won't be a case where this template creates by mistake which will result in a blueprint in the Template Store